### PR TITLE
Remove fiat currencies

### PR DESCRIPTION
**Pull request by slunak**
Here is my manual approach to remove fiat currencies.

### CODE REVIEW

**Comment by slunak** [comment]
@frederikbosch WDYT?

**Comment by frederikbosch** [comment]
I can only look at the code, not at the currencies itself as I lack the knowledge. I see you are doing a manual removal. If that other source you mentioned earlier has info on which currency is a fiat currency, I think we should go for that one in the future. It would make updating the repo easier. But for now, let's go for this solution.

File: src/Fetcher/BinanceFetcher.php [hunk 1]
```diff
@@ -49,4 +49,24 @@ public function fetch(): array
 
         return $cryptocurrencies;
     }
+
+    private function isFiatCurrency($symbol): bool
+    {
+        $fiatCurrencies = array(
```
Comment (frederikbosch) [inline src/Fetcher/BinanceFetcher.php]: This could be a class constant.

**Review by frederikbosch — APPROVED** [review]

File: src/Fetcher/BinanceFetcher.php (hunk 1 above)
Comment (slunak) [reply src/Fetcher/BinanceFetcher.php]: Thanks, updated the code.

**Comment by slunak** [comment]
Fixes #3

**Pull request merged**